src/foam/u2/view/RadioView.js

```diff
@@ -20,10 +20,27 @@ foam.CLASS({
   name: 'RadioView',
   extends: 'foam.u2.view.ChoiceView',
 
+  axioms: [
+    foam.u2.CSS.create({
+      code: function CSS() {/*
+        ^{ }
```

Comment:
I don't think you need to define an empty CSS class.  You can still apply this.myClass() to your elements and just have